Features/Lookuptable/testPages/data/lookup_table_page.py

```diff
@@ -0,0 +1,771 @@
+        self.valueerror = (By.XPATH,"//*[@class='fd-scrollable fd-props-scrollable']/form/fieldset/div/div/div[2]/div/div/div/div")
+        self.displayerror = (By.XPATH,"//*[@class='fd-scrollable fd-props-scrollable']/form/fieldset/div/div/div[3]/div/div/div")
+        self.autoselectedvalue = (By.XPATH,"//ul[@class='atwho-view-ul']/li[@class='cur'] ")
+        self.autoselectedvalue1 = (By.XPATH,"//*[@id='atwho-ground-property-labelRef']//ul[@class='atwho-view-ul']/li[@class='cur']")
+        self.edit_state_table = (By.XPATH,"(//tr[td[span[text()='state']]]//button)[1]")
+        self.select_column = (By.XPATH, "(//div[div[h4[span[text()='state']]]]/div[2]/fieldset/div[4]/table/tbody/tr/td[2]/input)[1]")
+        self.savestate = (By.XPATH, "//div[div[h4[span[text()='state']]]]//button[@data-bind='click: saveEdit']")
+        self.Ecase_list = (By.XPATH, "(//*[@data-label='Edit Pen'])[1]")
+        self.errormessage = (By.XPATH, "(//*[@class='messages'])[1]")
+        self.savedbutton = (By.XPATH, "//span[text()='Saved']")
+        self.numberofquestions = (By.XPATH, "//*[@id='formdesigner']/div[1]/div[1]/div[1]/div[4]/div[1]/ul/li")
+        self.childnode = (By.XPATH, "//*[@id='formdesigner']/div[1]/div[1]/div[1]/div[4]/div[1]/ul/li/ul")
+        self.home = (By.XPATH,"//*[@id='breadcrumb-region']/div/div/ol/li[1]")
+        self.sync = (By.XPATH,"//*[@class='ff ff-sync appicon-icon']")
+        self.label = (By.XPATH,"//*[@data-qtype='Trigger']")
+        self.settings = (By.XPATH, "//*[@class='fa fa-gear appicon-icon']")
+        self.app_language = (By.XPATH,"//*[@class='form-control js-lang']")
+        self.done =(By.XPATH, "//*[@class='btn btn-primary js-done']")
+        self.selected_caselist =(By.XPATH, "(//*[@class='appnav-item ']/a[@class='appnav-delete'])[1]")
+
+
+    def create_lookup_table(self):
+        self.wait_to_click(self.manage_tables_link)
+        self.wait_to_click(self.add_table)
+        self.send_keys(self.table_id, self.table_id_name)
+        self.send_keys(self.table_id_description, self.table_id_name)
+        self.wait_to_click(self.add_field)
+        self.send_keys(self.field_name, self.table_id_name)
+        self.wait_to_click(self.save_table)
+        time.sleep(2)
+        assert self.is_present_and_displayed(self.table_created_path)
+        print("LookUp Table created successfully!")
+        return self.table_id_name
+
+    def create_download_lookup_table_without_field(self):
+        self.wait_to_click(self.manage_tables_link)
+        self.wait_to_click(self.add_table)
+        self.send_keys(self.table_id, self.table_id_name)
+        self.send_keys(self.table_id_description, self.table_id_name)
+        self.wait_to_click(self.save_table)
+        time.sleep(2)
+        assert self.is_present_and_displayed(self.table_created_path)
+        print("LookUp Table created successfully!")
+        self.download1()
+        return self.table_id_name
+
+    def view_lookup_table(self,table_id_name):
```

Comment:
why are you sending table_id_name as a parameter but using self.table_id_name inside the function?